packages/client/tests/functional/field-reference/json/__snapshots__/tests.ts.snap

```diff
@@ -7,19 +7,9 @@ Invalid \`prisma.product.findMany()\` invocation in
 
   XX 
   XX // TODO: Edge: skipped because of the error snapshot
-  XX testIf(runtime !== 'edge' && getQueryEngineProtocol() !== 'json')('wrong field type', async () => {
-→ XX   const products = prisma.product.findMany({
-          where: {
-            properties1: {
-              equals: prisma.product.fields.title
-              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-            }
-          }
-        })
-
-Argument equals: Got invalid value prisma.product.fields.title on prisma.findManyProduct. Provided StringFieldRefInput<Product>, expected Json or JsonFieldRefInput or JsonNullValueFilter.
-
-
+  XX testIf(runtime !== 'edge')('wrong field type', async () => {
+→ XX   const products = prisma.product.findMany(
```

Comment:
we lost the highlighting of the property in question here

Reply:
Yea, that's known limitation of engine-side validation for that. Basically, this particular rule is validated in a completely different layer which does not have all the necessary metadata to send to the client for rendering for rendering anymore. Back in the day, we've decided that message is ok-ish to figure out what's going on.